packages/frontend/src/components/DataViz/config/DataVizAggregationConfig.tsx

```diff
@@ -30,6 +30,7 @@ const AggregationIcon: FC<{ aggregation: string | undefined }> = ({
             icon = IconSum;
             break;
         case MetricType.AVERAGE:
+        case VizAggregationOptions.AVERAGE:
             icon = IconTrendingUp;
```

Comment:
I don't think this is a great icon for `average` , but I can't find a find a better one to replace it

Reply:
[IconMathAvg](https://tabler.io/icons/icon/math-avg)

This apparently 😅

Reply:
You think ? that doesn't look right to me..

Reply:
I mean the name of the icon has avg, but I don't see it being widely used as **the** symbol for average. what you have works for me, checking the code!